src/backend/distributed/commands/index.c

```diff
@@ -921,3 +913,38 @@ DropIndexTaskList(Oid relationId, Oid indexId, DropStmt *dropStmt)
 
 	return taskList;
 }
+
+
+/*
+ * FinishConcurrentIndexCommand marks new indexes valid if they were created using
+ * the CONCURRENTLY flag, because we marked it as invalid in PostProcessIndexStmt
+ * to ensure failures leave behind an invalid index.
+ */
+void
+FinishConcurrentIndexCommand(IndexStmt *indexStmt)
+{
+	Assert(indexStmt->concurrent);
+	Assert(IsCoordinator());
+
+	/*
+	 * We make sure schema name is not null in the PreprocessIndexStmt
+	 */
+	Oid schemaId = get_namespace_oid(indexStmt->relation->schemaname, true);
```

Comment:
I see that we are sure that table/schema exists but giving missingOk to be false might a bit safer maybe